utility/mm2_crawler

```diff
@@ -135,7 +135,12 @@ def doit(options, config):
         if any(item.startswith('^') for item in options.continents):
             for item in options.continents:
                 if item.startswith('^'):
-                    continents.remove(item[1:])
+                    try:
+                        continents.remove(item[1:].upper())
+                    except ValueError:
+                        # A continent was specified which is not
```

Comment:
Yes, you are right. I thought the same when writing that part. I will shorty update this PR and then the output will be like this:

```
WARNING:crawler:Hosts(0/0):Threads(0/0):0:master:Ignoring unknown continent: E
WARNING:crawler:Hosts(0/0):Threads(0/0):0:master:Not in the list of known continents. ['AF', 'AN', 'AS', 'EU', 'NA', 'OC', 'SA', '--']
```